.travis.yml

```diff
@@ -38,7 +38,7 @@ script:
   - flake8 --version
   - flake8 src --extend-ignore=D  # ignore docstrings for now
   - flake8 --exclude=src,docs --extend-ignore=D
-  - pytest --cov=viskit
+  - pytest -n $(nproc) --cov=viskit
```

Comment:
I believe this can also just be `-n auto`, according to the pytest-xdist documentation.